device/mellanox/x86_64-mlnx_msn4600-r0/ACS-MSN4600/buffers_dynamic.json.j2

```diff
@@ -0,0 +1 @@
+../../x86_64-mlnx_msn2700-r0/ACS-MSN2700/buffers_dynamic.json.j2
```

Comment:
dont we have one for 4700? all are using 2700 only?

Reply:
All the SPC3 switched uses the 2700, including the 4700